tests/coreTest.php

```diff
@@ -94,4 +95,23 @@ function testSiteSelectionViaCwd() {
     }
     chdir($cwd);
   }
+
+  function testOptionsUri() {
+    // Put a yml file in the drush folder.
+    $drush_directory = sprintf('%s/drush', $this->webroot());
+    $drush_config_file = sprintf('%s/drush.yml', $drush_directory);
+    $test_uri = 'http://test.uri';
+    $options_with_uri = [
+      'options' => [
+        'uri' => $test_uri,
+      ],
+    ];
+    @mkdir($drush_directory);
```

Comment:
We usually use $this->mkdir()

Reply:
Ah thanks, did not know this. However, I removed this instead after re-using the root drush dir